From ec0a927527d8070984619de3b6254d198d307af9 Mon Sep 17 00:00:00 2001
From: Ricardo Salveti <ricardo@foundries.io>
Date: Tue, 8 Jan 2019 22:49:45 -0200
Subject: [PATCH] rpi: prefer downstream dtb files

Overlays are compatible only with the downstream dtb files, so prefer
downstream to avoid breaking rpi users.

Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
---
 board/raspberrypi/rpi/rpi.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/board/raspberrypi/rpi/rpi.c b/board/raspberrypi/rpi/rpi.c
index 617c892dde..e910d1e82f 100644
--- a/board/raspberrypi/rpi/rpi.c
+++ b/board/raspberrypi/rpi/rpi.c
@@ -115,7 +115,7 @@ static const struct rpi_model rpi_models_new_scheme[] = {
 	},
 	[0x8] = {
 		"3 Model B",
-		DTB_DIR "bcm2837-rpi-3-b.dtb",
+		DTB_DIR "bcm2710-rpi-3-b.dtb",
 		true,
 	},
 	[0x9] = {
@@ -125,17 +125,17 @@ static const struct rpi_model rpi_models_new_scheme[] = {
 	},
 	[0xA] = {
 		"Compute Module 3",
-		DTB_DIR "bcm2837-rpi-cm3.dtb",
+		DTB_DIR "bcm2710-rpi-cm3.dtb",
 		false,
 	},
 	[0xC] = {
 		"Zero W",
-		DTB_DIR "bcm2835-rpi-zero-w.dtb",
+		DTB_DIR "bcm2708-rpi-0-w.dtb",
 		false,
 	},
 	[0xD] = {
 		"3 Model B+",
-		DTB_DIR "bcm2837-rpi-3-b-plus.dtb",
+		DTB_DIR "bcm2710-rpi-3-b-plus.dtb",
 		true,
 	},
 	[0xE] = {
@@ -145,7 +145,7 @@ static const struct rpi_model rpi_models_new_scheme[] = {
 	},
 	[0x10] = {
 		"Compute Module 3+",
-		DTB_DIR "bcm2837-rpi-cm3.dtb",
+		DTB_DIR "bcm2710-rpi-cm3.dtb",
 		false,
 	},
 };
-- 
2.22.0

